src/vanilla/Model/CodeModelTS.cs

```diff
@@ -18,6 +18,11 @@ public class CodeModelTS : CodeModel
     {
         private const string ServiceClientOptions = "ServiceClientOptions";
 
+        private const string defaultGitHubRepositoryName = "azure-sdk-for-javascript";
```

Comment:
azure-sdk-for-js

Reply:
Done in https://github.com/Azure/autorest.typescript/pull/200/commits/f13760a76824c305882e4e0acb9a8085699a5666.